Tools/autotest/web-firmware/Tools/FilterTool/index.html

```diff
@@ -5,6 +5,8 @@
 <title>ArduPilot Filter Analysis</title>
 <script type="text/javascript" src="filters.js"></script>
 <script type="text/javascript" src="FileSaver.js"></script>
+<script type="text/javascript" src={{url_for('static', filename='filters.js')}}></script>
+<script type="text/javascript" src={{url_for('static', filename='FileSaver.js')}}></script>
```

Comment:
I had to add these includes for the new flask launcher, it doesn't seem to care that the original includes are still there. I'm not sure these will mess up the how its currently working on the server or not?